examples/domainmodel/src/cli/index.ts

```diff
@@ -12,6 +12,16 @@ import { languageMetaData } from '../language-server/generated/module';
 import { extractAstNode } from './cli-util';
 import { generateJava } from './generator';
 
+export const generateAction = (fileName: string, opts: GenerateOptions) => {
```

Comment:
There's a linter warning here. Please install the ESLint VS Code extension so you see such linter issues in the editor.

Reply:
Yeah, just had it deactivated.

Reply:
Btw. I refactored this action into the cli-utils, which is a bit more consequent because I extracted it for better testing/debugging purposes.